app/views/dashboards/_subject.html.erb

```diff
@@ -21,6 +21,19 @@
     </div>
   <% end %>
 
+<% if user.homepage_url.present? %>
+  <div class="flex items-center mb-4 text-b3 lg:text-b2">
+    <%= icon_tag("link", color: :primary, class: "w-6 text-orange mr-3") %>
+    <p class="text-neutral-800 dark:text-white"><%=
+      link_to(
+        truncate(user.homepage_url, length: 20),
```

Comment:
Taking into account really long homepage URLs. 
I chose length 20 because I saw that is the length we limit our Twitter username 

### Before: 
<img width="640" alt="Screenshot 2024-11-21 at 16 33 36" src="https://github.com/user-attachments/assets/d45dc2ef-1030-443c-ae84-b7179cfeaa25">


### After 
<img width="616" alt="Screenshot 2024-12-03 at 13 54 54" src="https://github.com/user-attachments/assets/02c43933-76e6-430a-9bc7-173fb5693197">
<img width="793" alt="Screenshot 2024-12-03 at 13 55 07" src="https://github.com/user-attachments/assets/3b4a9e77-595d-4061-aad8-bb06ff0bbb8f">